core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -2794,6 +2794,13 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
           .setScope(Scope.CLIENT)
           .build();
+  public static final PropertyKey USER_MULTI_RANGE_CHUNK_SIZE_BYTES =
+      new Builder(Name.USER_MULTI_RANGE_CHUNK_SIZE_BYTES)
+          .setDefaultValue(String.format("${%s}", Name.USER_BLOCK_SIZE_BYTES_DEFAULT))
```

Comment:
how does this default value work?

Reply:
It's default is set to whatever the `USER_BLOCK_SIZE_BYTES_DEFAULT` is. When formatted it would look like `${alluxio.user.block.size.bytes.default}`. In the case the property name every changed I thought it would be better to format and use the variable rather than specifically write `${alluxio.user.block.size.bytes.default}`